.github/workflows/hlint.yml

```diff
@@ -15,12 +15,12 @@ jobs:
     - name: 'Installing'
       uses: rwe/actions-hlint-setup@v1
       with:
-        version: '3.6.1'
+        version: '3.8'
 
     - name: 'Checking code'
       uses: rwe/actions-hlint-run@v2
       with:
-        hlint-bin: "hlint --with-group=extra"
+        hlint-bin: "hlint --with-group=extra --ignore-glob=**/testdata/** --ignore-glob=**/test/data/**"
```

Comment:
Can you leave a comment with a link to the hlint issue so we know to move this back into the config file when it gets fixed?

Reply:
I dig into the hlint source code and found this. There is no option to configure the ignore glob, 0 0. Also no issue in hlint about this.
```
        allowFields v ["lhs","rhs","note","name","side"]
        let hintRuleScope = mempty
        pure $
          Left HintRule {hintRuleSeverity = severity, hintRuleLHS = extendInstances lhs, hintRuleRHS = extendInstances rhs, ..}
            : [Right $ Classify severity hintRuleName "" "" | not isBuiltin]
     else do
        names <- parseFieldOpt "name" v >>= maybe (pure []) parseArrayString
        within <- parseFieldOpt "within" v >>= maybe (pure [("","")]) (parseArray >=> concatMapM parseWithin)
        pure [Right $ Classify severity n a b | (a,b) <- within, n <- ["" | null names] ++ names]
```

Reply:
`within` field is just about `severity`, which we mistakenly thought was about glob.

Reply:
The only source code I see about ignoring is 
```
        ignore = (toPredicate $ cmdIgnoreGlob cmd)
        toPredicate :: [FilePattern] -> FilePath -> Bool
        toPredicate [] = const False
        toPredicate globs = \x -> not $ null $ m [((), cleanup x)]
            where m = matchMany (map ((),) globs)
```
where cmdIgnoreGlob is only picked up by commandline arg

Reply:
I mean, didn't you do https://github.com/ndmitchell/hlint/pull/1554 ? So we can fix put this back once that is released?

Reply:
Oh that one is about apply the ignoring glob correctly, it is already included in the hlint 3.8. 
Before that, even with commandline ignoring glob, we won't be ignoring some files.

Reply:
https://github.com/ndmitchell/hlint/pull/1554 is the reason we need to bump up to 3.8